stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/ChannelsScreen.kt

```diff
@@ -152,9 +152,11 @@ public fun ChannelsScreen(
         if (activeAction is LeaveGroup) {
             SimpleDialog(
                 modifier = Modifier.padding(16.dp),
```

Comment:
A bit of duplication here, but that's basically how we want to do it: ensure that strings are customizable independently, for example, for "Leave group" button and a confirmation dialog with "Leave group" title.

Reply:
That makes sense to me and I'm happy with it - if by design we want to change what these are, we can do it separately! :]